platforms/core-configuration/model-reflect/src/main/java/org/gradle/internal/reflect/validation/DelegatingProblemBuilder.java

```diff
@@ -126,11 +126,6 @@ public <T extends AdditionalData> InternalProblemBuilder additionalData(Class<T>
         return validateDelegate(delegate.additionalData(type, config));
     }
 
-    @Override
```

Comment:
Your changes already moved all additionalData definitions to use the public API, right?

Reply:
yes and the other internal API left.